docs/Tutorials/Syntax/Quest-Packages.md

```diff
@@ -0,0 +1,360 @@
+       startedTag: "tag startedTag"
+     ```
+
+You can download this example as well. No worries, the files of the previous example will not get overwritten.
+Instead, a new folder will be created in the previous package.
+
+--8<-- "Tutorials/download-complete-files.md"
+    ```
+    /bq download BetonQuest/Quest-Tutorials main QuestPackages /Syntax/packageStructure/2-SingleFileStructure /packageStructure/SingleFile
+    ```
+    You can now find all files needed for this tutorial in this location:
+    "_YOUR-SERVER-LOCATION/plugins/BetonQuest/QuestPackages/packageStructure/SingleFile_"
+
+## 4. Creating packages in packages
+
+It is also possible to have a package inside a package.
+And it's pretty easy to do! Just look at the following structure:
+
+<div class="grid" markdown>
+!!! example "Example 1"
+    * :material-folder-open: QuestPackages
+        - :material-folder-open: woodCuttingQuest
+            - :material-file-star: package.yml
+            - {==:material-folder-open: miningQuest==}
+                - {==:material-file-star: package.yml==}
+
+!!! example "Example 2"
+    * :material-folder-open: QuestPackages
+        - :material-folder-open: woodCuttingQuest
+            - :material-file-star: package.yml
+              - {==:material-folder-open: miningQuest==}
+                - {==:material-file-star: package.yml==}
+                  - {==:material-folder-open: fishingQuest==}
+                    - {==:material-file-star: package.yml==}
+</div>
+
+Not hard to understand but to make things clearer:
+Every folder that contains a _package.yml_ file is a standalone quest package.
+By creating a folder with a _package.yml_ file inside another package, you create a standalone "subpackage". 
+
+--8<-- "Tutorials/download-complete-files.md"
+    ```
+    /bq download BetonQuest/Quest-Tutorials main QuestPackages /Syntax/packageStructure/3-PackageInPackage /packageStructure/PackageInPackage
+    ```
+    You can now find all files needed for this tutorial in this location:
+    "_YOUR-SERVER-LOCATION/plugins/BetonQuest/QuestPackages/packageStructure/PackageInPackage_"
+
+## 5. Creating subdirectories in a package
+
+Now you know, that it is possible to do nearly everything to structure your quests,
+But there is one more possibility: _subdirectories_.
+Let me first show you the example for it:
+
+!!! success "Example"
+    * :material-folder-open: QuestPackages
+        - :material-folder-open: woodCuttingQuest
+            - :material-file: package.yml
+            - {==:material-folder-open: questPart1==}
+                - :material-file: myEventsPart1.yml
+                - :material-file: MageConversation.yml
+            - {==:material-folder-open: questPart2==}
+                - :material-file: myEventsPart2.yml
+                - :material-file: KingConversation.yml
+
+You can create as many subdirectories in your quest packages as you like.
+Just create a folder with a name of your choice and start adding files to it!
```

Comment:
I would make clearer, that all sub folders belog to the package, as long as no new package.yml file is found